consensus/hotstuff/notifications/log_consumer.go

```diff
@@ -50,32 +54,65 @@ func (lc *LogConsumer) OnDoubleProposeDetected(block *model.Block, alt *model.Bl
 		Msg("double proposal detected")
 }
 
-func (lc *LogConsumer) OnReceiveVote(currentView uint64, vote *model.Vote) {
+func (lc *LogConsumer) OnReceiveProposal(currentView uint64, proposal *model.Proposal) {
+	lc.logBasicBlockData(lc.log.Debug(), proposal.Block).
+		Uint64("cur_view", currentView).
```

Comment:
taking a look at [`logBasicBlockData`](https://github.com/onflow/flow-go/blob/d74cd32abc0770f437da1b22c9e0e98ffeda662e/consensus/hotstuff/notifications/log_consumer.go#L177-L189): 
I don't think we log any information about the TC. But I think this would be very helpful. Could you add such logging

Reply:
I will extend logging for `OnReceiveProposal` but can't do it for `logBasicBlockData` since it accepts `Block` not `Proposal`. Unless we change `OnBlockIncorporated` and `OnFinalizedBlock` to work with proposals.